CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+### Default owners ###
+
+*                                           @godaddy/gasket-admins
+
+### Package maintainers ###
+
+/packages/gasket-cli/*                      @indexzero, @kinetifex
+/packages/gasket-engine/*                   @DullReferenceException
```

Comment:
I know this well enough to be listed here. I contributed `execApply` which really made me have to understand the symphony of thunks it uses.